packages/ripple-nuxt-tide/lib/core/tide-helper.js

```diff
@@ -124,7 +126,9 @@ export const getPageHeadConfig = ({
       { name: 'twitter:image:alt', hid: 'hid:image:alt', content: imageAlt },
       // Custom page meta
       { name: 'sitesection', content: siteSectionName },
-      { name: 'content-type', content: pageType && pageType.replace('node--', '') }
+      { name: 'content-type', content: pageType && pageType.replace('node--', '') },
+      // Robots meta
+      { name: 'robots', content: robotsMeta }
```

Comment:
If no robots content, should we just not add it? @MdNadimHossain

Reply:
@tim-yao yes, the check was there but then I removed it thinking empty check at `page-head.js` will not add it, but yeah I have added the check again here like this - `robotsMeta ? { name: 'robots', content: robotsMeta } : ''`